onnxruntime/core/optimizer/avx2_weight_s8_to_u8.cc

```diff
@@ -0,0 +1,293 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if (defined(_M_AMD64) && !defined(_M_ARM64EC)) || defined(_M_IX86) || defined(__x86_64__) || defined(__i386__) || !defined(DISABLE_CONTRIB_OPS)
+
+#include "core/optimizer/avx2_weight_s8_to_u8.h"
+
+#include <algorithm>
+
+#include "core/graph/graph_utils.h"
+#include "core/optimizer/initializer.h"
+#include "core/optimizer/utils.h"
+
+namespace onnxruntime {
+
+/**
+ * @brief Convert the source int8_t TensorProto to a uint8_t one if the tensor contains
+ *        value outside of [-64, 63]
```

Comment:
>63 [](http://example.com/codeflow?start=33&length=2)

64 is also good. ORT quant tool uses range [-64, 64] for reduced range.

Reply:
fixed